### ISSUE TITLE
to_latex(as_document=False) should not wrap output in figure env

### ISSUE THREAD

**Issue by dimpase** [networkx/networkx#6357]
### Current Behavior

`tikzpicture` wrapped in `figure` environment
```
>>> print(nx.to_latex(gn, as_document=False))
\begin{figure}
  \begin{tikzpicture}
    ...
    \end{tikzpicture}
\end{figure}
```

### Expected Behavior

There is no extra environment around `tikzpicture`. Indeed, the latter may be used without an extra wrapper, or there can be
several `tikzpicture` in a `figure`. Literal programming systems such as [SageTeX](https://ctan.org/pkg/sagetex) provide ways to invoke Python from TeX macros, e.g. something like the following
```tex
\documentclass{article}
\usepackage{tikz} 
\usepackage{sagetex} 

\begin{document}

\begin{sagesilent}
 import networkx as nx
 g=graphs.PetersenGraph().networkx_graph()
\end{sagesilent}

Here is my favourite graph:
\sagestr{nx.to_latex(g, as_document=False)}

\end{document}
``` 
should, after processing, give a pdf with the picture of Petersen graph after the text,
not placed somewhere `figure` thinks is appropriate.

### Steps to Reproduce

Take your favourite graph as `gn` and call `print(nx.to_latex(gn, as_document=False))`

### Environment

Python version: 3.9.2
NetworkX version:  3.0

**Comment by dimpase** [comment]
using `figure_wrapper=''` suppresses all output, so this is not a way to deal with the issue here.

**Comment by dschult** [comment]
Did you try `nx.to_latex_raw`?  I'm not sure if that's what you want though...

**Comment by dimpase** [comment]
Actually, tikz(picture) understands radians, like this: `(0.628r:10)` rather than just `(0.628:10)`.
So it's a simple fix.

**Comment by dschult** [comment]
I'm going to mark this as closed based on #6360.
Thanks so much for the help with a much better default layout scheme.
